### PR TITLE
feat: user usage gql & test case improve

### PR DESCRIPTION
**Pull request by darkskygit**

### CODE REVIEW

**Comment by darkskygit** [comment]
Current dependencies on/for this PR:
* canary
  * **PR #5074** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5074?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #5075** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5075?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #5076** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5076?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
        * **PR #5087** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5087?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #5142** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5142?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #5178** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/5178?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/toeverything/AFFiNE/5076?utm_source=stack-comment).

File: packages/backend/server/tests/auth.e2e.ts [hunk 1]
```diff
@@ -52,6 +58,13 @@ test.beforeEach(async t => {
   t.context.app = app;
   t.context.auth = auth;
   t.context.mail = mail;
+
+  // init features
+  const run = module.get(RunCommand);
```
Comment (forehalo) [inline packages/backend/server/tests/auth.e2e.ts]: we should not involve data migration in testing, just write test cases for current code branches and manually create old data if backwards compatibility test required

tests for data migration is another topic

Comment (darkskygit) [reply packages/backend/server/tests/auth.e2e.ts]: now only run feature config import

Comment (forehalo) [reply packages/backend/server/tests/auth.e2e.ts]: it's still too verbose, we need a convenient way to init everything including db before each tests

**Review by Brooooooklyn — APPROVED** [review]

**Pull request merged**